python/orca/example/learn/pytorch/brainMRI/brainMRI.py

```diff
@@ -127,96 +129,64 @@ def optim_creator(model, config):
                           momentum=config.get("momentum", 0.9))
     return optimizer
 
+
 def denormalize(images):
     means = torch.tensor([0.485, 0.456, 0.406]).reshape(1, 3, 1, 1)
     stds = torch.tensor([0.229, 0.224, 0.225]).reshape(1, 3, 1, 1)
     return images * stds + means
 
-def show_batch(dl):
-    for images, masks in dl:
-        fig1, ax1 = plt.subplots(figsize=(24, 24))
-        ax1.set_xticks([]); ax1.set_yticks([])
-        denorm_images = denormalize(images)
-        ax1.imshow(make_grid(denorm_images[:13], nrow=13).permute(1, 2, 0).clamp(0,1))
-        
-        fig2, ax2 = plt.subplots(figsize=(24, 24))
-        ax2.set_xticks([]); ax2.set_yticks([])
-        ax2.imshow(make_grid(masks[:13], nrow=13).permute(1, 2, 0).clamp(0,1))
-        break
-        
 
 parser = argparse.ArgumentParser(description='PyTorch brainMRI Example')
 parser.add_argument('--cluster_mode', type=str, default="local",
-                    help='The cluster mode, such as local, yarn-client, yarn-cluster, spark-submit or k8s.')
+                    help='The cluster mode, such as local, yarn-client, or spark-submit.')
 parser.add_argument('--backend', type=str, default="torch_distributed",
-                    help='The backend of PyTorch Estimator; '
-                        'bigdl, torch_distributed and spark are supported')
+                    help='The backend of PyTorch Estimator; torch_distributed and spark are supported')
 parser.add_argument('--batch_size', type=int, default=64, help='The training batch size')
 parser.add_argument('--epochs', type=int, default=2, help='The number of epochs to train for')
-parser.add_argument('--data_dir', type=str, default="./data", help='The path to dataset')
-parser.add_argument('--download', type=bool, default=False, help='Download dataset or not')
-parser.add_argument("--executor_memory", type=str, default="5g", help="executor memory")
-parser.add_argument("--driver_memory", type=str, default="5g", help="driver memory")
+parser.add_argument('--data_dir', type=str, default='./kaggle_3m', help='the path of the dataset')
+parser.add_argument('--additional_archive', type=str, default="kaggle_3m.zip#kaggle_3m", help='the zip dataset')
+parser.add_argument('--model_dir', type=str, default=os.getcwd(),
+                    help="the model save dir for spark backend")
 args = parser.parse_args()
 
 if args.cluster_mode == "local":
-    init_orca_context(memory="4g")
-elif args.cluster_mode.startswith("yarn"):
-    if args.cluster_mode == "yarn-client":
-        init_orca_context(cluster_mode="yarn-client")
-    elif args.cluster_mode == "yarn-cluster":
-        init_orca_context(cluster_mode="yarn-cluster", memory=args.executor_memory, driver_memory=args.driver_memory)
+    init_orca_context(memory='4g')
+elif args.cluster_mode.startswith('yarn'):
```

Comment:
`args.cluster_mode == "yarn-client"` ?    Becuase yarn-cluster is not supported

Reply:
It is somewhat reasonable to assume startswith-yarn == yarn-client.
We have said yarn-client in readme.